feed/index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Feed</title>
+    <link rel="stylesheet" href="/dist/css/index.css" />
+
+    <style>
+      .row {
```

Comment:
In this case, you can use the Bootstrap gutter class. See https://getbootstrap.com/docs/5.3/layout/gutters/#horizontal-gutters

And aside from that, using inline styling is against the making criteria.

Reply:
I solved the issue, used <style> because the website had scrollbar at the bottom. The issue was the "row" was not wrapped in a container.